pkg/daemon/constants.go

```diff
@@ -14,6 +14,10 @@ const (
 	// MachineConfigDaemonStateDegraded is set by daemon when update cannot be applied.
 	MachineConfigDaemonStateDegraded = "Degraded"
 
+	// This label was introduced in addition to state/Degraded because we
+	// want operators and admins to be able to easily query this state.
+	MachineConfigDaemonLabelDegraded = "machineconfiguration.openshift.io/degraded"
```

Comment:
How about just making this `machineconfiguration.openshift.io/state` and have the label value be the state, just like the annotation? That way we can also easily query for `Working` and `Done` too.

Reply:
Well at that point, why have both annotations and labels?  Seems like you're saying we should basically switch from annotations to labels?  That could make sense.  I'd want the opinion of @abhinavdahiya for that.

Reply:
My thought on this though is that an admin (or operator) that wants to know about working state should be querying the `machineconfigpool`, not looking at nodes.

IOW, the working/done annotations are mostly an implementation detail of `machineconfigpool`.  But degraded is a much bigger deal and something that e.g. the machine-api-operator should be involved with.

Reply:
Don't add labels. Labels have greater meaning in kubernetes. Only annotations should be used.